fiftyone/utils/eval/segmentation.py

```diff
@@ -495,7 +509,7 @@ def _from_dict(cls, d, samples, config, eval_key, **kwargs):
         )
 
     @staticmethod
-    def _parse_confusion_matrix(confusion_matrix, classes):
+    def _parse_confusion_matrix(confusion_matrix, classes, ytrue_ids_dict, ypred_ids_dict):
```

Comment:
What's the purpose of this? You don't seem to use the new args in the method.

Reply:
Fixed it, those were remaining from my previous commit.